pandas/tests/reductions/test_reductions.py

```diff
@@ -787,6 +787,15 @@ def test_mean_masked_overflow(self):
         assert result_masked - result_numpy == 0
         assert result_masked == 1e17
 
+    def test_var_masked_array(self):
```

Comment:
Could you parameterize over `ddof=[0, 1]`?

Reply:
added